roles/java/meta/argument_specs.yml

```diff
@@ -0,0 +1,88 @@
+---
+argument_specs:
+  main:
+    short_description: Main entrypoint for the java role
+    options:
+      java_keystore:
+        type: dict
+        options:
+          path:
+            description: |
+              Local path to the java keystore
+            required: true
```

Comment:
probably not, if user doesn't requires mtls

Reply:
This use case was declined in https://alfresco.atlassian.net/wiki/spaces/TECH/pages/1124696953/Certificate+deployment+in+Ansible+playbook